python/paddle/tensor/math.py

```diff
@@ -7056,6 +7056,81 @@ def ldexp_(x, y, name=None):
     return paddle.multiply_(x, paddle.pow(two, y))
 
 
+def copysign(x, y, name=None):
+    r"""
+    Create a new floating-point tensor with the magnitude of input and the sign of other, elementwise.
+
+    Equation:
+        .. math::
+
+            copysign(x_{i},y_{i})=\left\{\begin{matrix}
+            & -|x_{i}| & if \space y_{i} < 0.0\\
```

Comment:
should be `y_{i} <= -0.0`? example code below provide cases when `y_{i} == 0.0` and `y_{i} == -0.0` to facilitate user understanding

Reply:
> should be `y_{i} <= -0.0`? example code below provide cases when `y_{i} == 0.0` and `y_{i} == -0.0` to facilitate user understanding

paddle seems doesn't specifically identify the `+0.0` and `-0.0`?

Reply:
> > should be `y_{i} <= -0.0`? example code below provide cases when `y_{i} == 0.0` and `y_{i} == -0.0` to facilitate user understanding
> 
> paddle seems doesn't specifically identify the `+0.0` and `-0.0`?

Leave it alone, there's a  bug/difference between paddle and pytorch when I commented before, maybe it's about the magic function of `-` operator.

In paddle:
```python
>>> import paddle
>>> paddle.to_tensor([0.0])
Tensor(shape=[1], dtype=float32, place=Place(cpu), stop_gradient=True,
       [0.])
>>> x = paddle.to_tensor([0.0])
>>> x = -x
>>> x
Tensor(shape=[1], dtype=float32, place=Place(cpu), stop_gradient=True,
       [0.])
>>>
```

In pytorch
```python
>>> import torch
>>> x = torch.tensor([0.])
>>> x
tensor([0.])
>>> -x
tensor([-0.])
>>>
```

Reply:
there are different issues:
1. The sign bit of `0.0` and `-0.0` are different, so `copysign(x, 0.0)` and `copysign(x, -0.0)` will be different, it is necessary to inform the user of their differences
2. when `x=0.0`, -x should be -0.0, this is a bug. If you are interested, you can propose another PR correction

Reply:
> there are different issues:
> 1. The sign bit of `0.0` and `-0.0` are different, so `copysign(x, 0.0)` and `copysign(x, -0.0)` will be different, it is necessary to inform the user of their differences between `y_{i} = 0.0` and `y_{i} = -0.0`, this is the reason why the suggestion above judgment condition is best set to `y_{i} <= -0.0`
> 2. when `x=0.0`, -x should be -0.0, this is a bug. If you are interested, you can propose another PR correction

Got it.